components/automate-ui/src/app/pages/+compliance/+reporting/+reporting-node/reporting-node.component.ts

```diff
@@ -30,7 +32,7 @@ export class ReportingNodeComponent implements OnInit, OnDestroy {
   constructor(
     private route: ActivatedRoute,
     private statsService: StatsService,
-    private reportQuery: ReportQueryService,
+    private reportQueryService: ReportQueryService,
```

Comment:
I renamed this service in a few files from reportQuery to reportQueryService to be more reflective of what it is.  There is also a ReportQuery object and type - having reportQuery sometimes be the service and sometimes be an object was getting confusing.